### ISSUE TITLE
pretraining procedure?

### ISSUE THREAD

**Issue by andrewyu90** [mapooon/BlendFace#15]
Hi, it seems this work, the pretraining scheme, is very important.

How can I do the pre-training?

**Comment by mapooon** [comment]
Hi, 
Please see the instruction for pre-training [here](https://github.com/mapooon/BlendFace/tree/master/training).

**Comment by andrewyu90** [comment]
Hey, thanks for a very, very quick answer. 

So, the training parts are actually for the 'pre-training'.

How about the training for the entire 'face swapping'? 

it seems like there is no code to compute the total loss (L_adv  + L_mask + lambda_1*L_id + lambda_2*L_recon + lambda_3*L_cycle.

**Comment by mapooon** [comment]
> So, the training parts are actually for the 'pre-training'.

Yes!

>How about the training for the entire 'face swapping'?

We do not plan to release the training code at this time. 
But our training scheme is based on an unofficial implementation of AEI-Net [here](https://github.com/maum-ai/faceshifter).
So, you can implement our model by modifying the loss functions.
Also, you can choose different backbones like [SimSwap](https://github.com/neuralchen/SimSwap).

**Comment by andrewyu90** [comment]
Thanks for the answering
